spec/acceptance/suites/default/lib/util.rb

```diff
@@ -18,46 +18,63 @@ module TlogTestUtil
   # @param timeout [Integer]
   #   Session timeout in seconds
   #
+  # @param command [String]
+  #   The command to run
+  #
   # @return [Hash]
   #   :success [Boolean] => Whether or not the command was successful
   #   :output [String]   => The output from the session
   def local_ssh(host, port, user, password, timeout=5)
     require 'timeout'
-    require 'pty'
-    require 'expect'
+    require 'net/ssh'
 
     to_return = {
       :success => false,
       :output  => []
     }
 
+    ssh_opts = {
+      # Ignore ssh-agent
+      :keys_only             => true,
+      :non_interactive       => true,
+      :password              => password,
+      :port                  => port,
+      :timeout               => timeout,
+      :user_known_hosts_file => ['/dev/null'],
+      :verify_host_key       => :never,
+      # For FIPS testing
+      :encryption            => 'aes256-ctr',
+      :hmac                  => ['hmac-sha2-256', 'hmac-sha1']
+    }
+
     begin
-      Timeout::timeout(timeout) do
-        # Cipher and HMAC set for the FIPS tests
-        PTY.spawn("ssh -tt -c aes256-ctr -m hmac-sha2-256,hmac-sha1 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -l #{user} -p #{port} #{host}") do |r, w, pid|
-          begin
-            r.expect(/password: /i) { |msg| w.puts(password) }
+      Net::SSH.start(host, user, ssh_opts) do |ssh|
```

Comment:
The method's comments mention that _"This is useful for capturing output messages that cannot otherwise consistently be accessed via Net::SSH,_" but this patch switches over to using Net::SSH—why is that consistent now?

Reply:
@op-ct Forgot to update the message. The native SSH method had issues with large keyrings as well as differences in underlying OS implementations as the releases grind forward. I'll update the comments.